src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = days * 40;
+  const minTwenty = 20;
```

Comment:
```suggestion
  const smallDiscount = 20;
```
Недоречно називати змінні тими ж іменами, які в них містяться значення. А якщо розмір малої знижки буде не 20, а 30? Тоді назва minTwenty буде зовсім не відповідати